pkg/kev/converter/kubernetes/project_service_test.go

```diff
@@ -1254,712 +1240,659 @@ var _ = Describe("ProjectService", func() {
+					Expect(result.HTTPGet.Port.IntValue()).To(Equal(8080))
+					Expect(result.HTTPGet.Path).To(Equal("/status"))
 				})
 			})
 
 			Context("with missing path", func() {
 				BeforeEach(func() {
-					labels = composego.Labels{
-						config.LabelWorkloadLivenessProbeType:     config.ProbeTypeHTTP.String(),
-						config.LabelWorkloadLivenessProbeHTTPPort: "8080",
-					}
-				})
-
-				It("returns an error", func() {
-					_, err := projectService.livenessHTTPProbe()
-					Expect(err).NotTo(BeNil())
-					Expect(err.Error()).To(ContainSubstring("%s not correctly defined", config.LabelWorkloadLivenessProbeHTTPPath))
-				})
-			})
-		})
-	})
-
-	Describe("livenessProbeTCP", func() {
-		When("defined via labels", func() {
-
-			Context("and supplied as string port", func() {
-				port := "8080"
-
-				BeforeEach(func() {
-					labels = composego.Labels{
-						config.LabelWorkloadLivenessProbeType:    config.ProbeTypeTCP.String(),
-						config.LabelWorkloadLivenessProbeTCPPort: port,
-					}
-				})
-
-				It("returns label value", func() {
-					p, err := projectService.livenessTCPProbe()
-					Expect(err).To(Succeed())
-					Expect(p.Port.String()).To(Equal(port))
-				})
-			})
-
-			Context("and empty port", func() {
-				BeforeEach(func() {
-					labels = composego.Labels{
-						config.LabelWorkloadLivenessProbeType:    config.ProbeTypeTCP.String(),
-						config.LabelWorkloadLivenessProbeTCPPort: "",
-					}
-				})
-
-				It("returns an error", func() {
-					p, err := projectService.livenessTCPProbe()
-					Expect(err).To(HaveOccurred())
-					Expect(err.Error()).To(Equal(fmt.Sprintf("%s needs to be a number", config.LabelWorkloadLivenessProbeTCPPort)))
-					Expect(p).To(BeNil())
-				})
-			})
-
-			Context("with NaN port", func() {
-				BeforeEach(func() {
-					labels = composego.Labels{
-						config.LabelWorkloadLivenessProbeType:    config.ProbeTypeTCP.String(),
-						config.LabelWorkloadLivenessProbeTCPPort: "asds",
-					}
+					k8sconf.Workload.LivenessProbe.Type = config.ProbeTypeHTTP.String()
+					k8sconf.Workload.LivenessProbe.HTTP.Port = 8080
 				})
 
 				It("returns an error", func() {
-					_, err := projectService.livenessTCPProbe()
-					Expect(err).NotTo(BeNil())
-					Expect(err.Error()).To(ContainSubstring("%s needs to be a number", config.LabelWorkloadLivenessProbeTCPPort))
-				})
-			})
-
-			Context("and no port label", func() {
-				BeforeEach(func() {
-					labels = composego.Labels{
-						config.LabelWorkloadLivenessProbeTCPPort: config.ProbeTypeTCP.String(),
-					}
-				})
-
-				It("returns an error", func() {
-					p, err := projectService.livenessTCPProbe()
-					Expect(err).To(HaveOccurred())
-					Expect(err.Error()).To(Equal(fmt.Sprintf("%s needs to be a number", config.LabelWorkloadLivenessProbeTCPPort)))
-					Expect(p).To(BeNil())
-				})
-			})
-		})
-
-	})
-
-	Describe("livenessProbeCommand", func() {
-		When("defined via labels", func() {
```

Comment:
There is a bunch of probe related tests being removed in this PR that set the specific contexts around various probe parameters being supplied via configuration (previously labels and now by extensions) and cases for when custom configuration isn't explicitly specified. Do we have the coverage of those things still?

Reply:
Yes, we now start with defaults and merge in the set configuration. There are some bits missing in terms of validation (https://github.com/appvia/kev/issues/460) for example 0 values and such but absent values are no longer a problem (and is tested in the config package), same as wrong types